aip/0132.md

```diff
@@ -131,8 +132,13 @@ request message **should** contain a `string order_by` field.
   `"foo,bar"`.
 - The default sorting order is ascending. To specify descending order for a
   field, users append a `" desc"` suffix; for example: `"foo desc, bar"`.
-- Redundant space characters in the syntax are insignificant. `"foo, bar desc"`
-  and `" foo , bar desc "` are equivalent.
+- Redundant space characters in the syntax are insignificant.
+  `"foo, bar desc"`, `" foo , bar desc "`, and `"foo,bar desc"` are all
+  equivalent.
+- Subfields are specified with a `.` character, such as `foo.bar` or
+  `address.street`.
+- The method **should** document the default ordering, if one exists, or
+  **should** explicitly document that the ordering behavior is unspecified.
```

Comment:
In cases where the ordering is not documented, should we convey something saying that consumers should expect that the order is random?

Reply:
That is what this bullet is intended to convey.